libmozdata/hgmozilla.py

```diff
@@ -241,3 +246,41 @@ def get(paths, channel='nightly', node='tip'):
             FileInfo(queries=queries).wait()
 
         return data
+
+
+class HGMozilla(object):
```

Comment:
Sure, it was in my idea for a next step: just have a factory to choose the way to get info from hg repo and off course have the same interface.